src/java/org/apache/poi/ss/formula/functions/AggregateFunction.java

```diff
@@ -257,4 +258,16 @@ protected double evaluate(double[] values) throws EvaluationException {
             return StatsLib.varp(values);
         }
     };
+    public static final Function GEOMEAN = new AggregateFunction() {
+        @Override
+        protected double evaluate(double[] values) throws EvaluationException {
+        	// The library implementation returns 0 for an input sequence like [1, 0]. So this check is necessary.
+        	for (double value: values) {
+        		if (value <= 0) {
+        			throw new EvaluationException(ErrorEval.NUM_ERROR);
```

Comment:
could you fix the indents in this file to be spaces  (not tabs)?

Reply:
fixed